src/tests/unit/doughs-dao.test.js

```diff
@@ -179,4 +184,52 @@ describe('test doughs dao', () => {
       });
     });
   });
+  context('getDoughById', () => {
+    let result;
+    beforeEach(() => {
+      connectionStub = sinon.stub().returns(executeReturn);
+      doughsDao = proxyquireDoughsDao(
+        connectionStub,
+        serializeDoughsStub,
+        serializeDoughStub,
+      );
+    });
+    context('when called with an integer id', () => {
```

Comment:
Looks like `context` is just an alias of `describe`. I'm fine using either one, but I think we should use the same one across all of the testings for the consistency.

Reply:
Technically, I think [this is the more idiomatic usage](https://lmws.net/describe-vs-context-in-rspec) of context and describe, and I was going to update future tests to do it this way. But I also don't feel that strongly about it and would be fine changing back if that would be easier to follow.

Reply:
Good to know. I'll follow this in the future as well 👍